### PR TITLE
[General] update "uninstall apps" + "deactivate users"

### PR DESCRIPTION
**Pull request by jcs-odoo**
task
https://www.odoo.com/web#id=2575136&action=333&active_id=3835&model=project.task&view_type=form&cids=1&menu_id=4720

### CODE REVIEW

File: content/applications/general.rst [hunk 1]
```diff
@@ -16,130 +16,70 @@ General
...
-Apps** to access the list of your installed applications.
-
-+----------------+----------------+
-|   |settings|   | |browse_apps|  |
-+----------------+----------------+
-
-In your applications' dashboard, you will see all the icons of your
-applications. Click on the application you want to uninstall. Then, on
-the form of the application, click on **Uninstall**.
+**Odoo** is a suite of open source business apps that cover all your company needs: :doc:`CRM
```
Comment (xpl-odoo) [inline content/applications/general.rst:19]:
```suggestion
**Odoo** is a suite of open source business apps covering all your company needs: :doc:`CRM
```

File: content/applications/general.rst [hunk 2]
```diff
@@ -16,130 +16,70 @@ General
...
+:doc:`project management <services/project>`, etc. Its unique value proposition is to be at the same
+time *very easy* to use and *fully integrated*.
+
+This section of the documentation covers all functional aspects that aren't app-specific. You can
+find, for example, information about:
+
+- :doc:`Users settings <general/users>` and their :doc:`access rights <general/users/access_rights>`
+- :doc:`Export and import data <general/export_import_data>`
+- :doc:`Payment Acquirers <general/payment_acquirers>` allow your customers to pay you with their
+  favorite payment method.
```
Comment (xpl-odoo) [inline content/applications/general.rst:31]:
```suggestion
- :doc:`Payment Acquirers <general/payment_acquirers>`, to allow your customers to pay you with their
  favorite payment method
```
If you read it the full sentence: "_You can find, for example, information about: Payment Acquirers allow your customers to pay you with their favorite payment method._", it feels weird. With the added comma and "to", it flows better. I would also remove the full stop at the end of that bullet point to be consistent with other bullet points.

File: content/applications/general.rst [hunk 3]
```diff
@@ -16,130 +16,70 @@ General
...
+
+.. important::
+   - | **Installing or uninstalling apps, managing users, etc., is up to you.**
+     | As the administrator of your database, you are responsible for its usage, as you know best
+       how your organization works. This includes the installation or uninstallation of :ref:`apps
+       <general/apps-modules>` as well as the number of :doc:`users <general/users>` currently in
+       use.
+   - | **Odoo Apps have dependencies.**
+     | This means that you may have to install modules that you do not actively use to access some
+       features of Odoo you might need.
```
Comment (xpl-odoo) [inline content/applications/general.rst:42]:
```suggestion
     | This means that you may have to install modules that you do not actively use to access features
       that are related to them.
```
I tried rephrasing to make it a bit clearer. Not sure I was successful though :)

File: content/applications/general.rst [hunk 4]
```diff
@@ -16,130 +16,70 @@ General
...
+   - | **Always test app installation/removal on a duplicate of your database.**
+     | This way, you can know what apps dependencies may be required or what data may be erased.
```
Comment (xpl-odoo) [inline content/applications/general.rst:44]:
```suggestion
     | This way, you can know what app dependencies may be required or what data may be erased.
```
I think app should be singular here.

File: content/applications/general.rst [hunk 5]
```diff
@@ -16,130 +16,70 @@ General
...
+
+.. seealso::
+   - :ref:`Online Database management: Duplicate a database <duplicate_online>`
+   - :ref:`On-premises Database management: Duplicate a database <duplicate_premise>`
+   - `Submit a Support Ticket <https://www.odoo.com/help>`_.
```
Comment (xpl-odoo) [inline content/applications/general.rst:49]:
```suggestion
   - `Submit a Support Ticket <https://www.odoo.com/help>`_
```
I would remove the full stop - although we could definitely argue about the proper formatting of bullet points' commas and full stops :)

File: content/applications/general.rst [hunk 6]
```diff
@@ -16,130 +16,70 @@ General
...
+Applications and modules
+========================
+
+Odoo is *not a smartphone*, and its apps shouldn't be installed or uninstalled carelessly. Apply
+caution when adding or removing applications and modules on your database since this may impact your
+subscription costs.
+
+.. _general/uninstall-apps:
+
+Uninstall Apps
```
Comment (xpl-odoo) [inline content/applications/general.rst:62]:
```suggestion
Uninstall Apps
```
Sometimes you capitalize Apps, sometimes not. Not sure what's best but let's aim for consistency right? :)

File: content/applications/general/users.rst [hunk 7]
```diff
@@ -62,6 +62,18 @@ The *Portal* and *Public* options do not allow you to choose access rights. Memb
 ones (such as record rules and restricted menus) and usually do not belong to the usual Odoo
 groups.
 
+.. _users/deactivate:
+
+Deactivate Users
+================
+
+Go to :menuselection:`Settings --> Users & Companies --> Users`, open the user you want to deactivate, click on
+*Action*, and then on *Archive*.
```
Comment (xpl-odoo) [inline content/applications/general/users.rst:71]: 100th character break

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Done :)

File: content/applications/general.rst (hunk 6 above)
Comment (jcs-odoo) [reply content/applications/general.rst:62]: You're right, it should be "apps" here.
It is "Apps" when referring to the "Apps" button in the main menu.

Comment (jcs-odoo) [reply content/applications/general.rst:62]: Also, I think that I should replace some "applications" with "apps", for consistency

File: content/applications/general.rst (hunk 3 above)
Comment (jcs-odoo) [reply content/applications/general.rst:42]: What do you think of this @xpl-odoo ?
```suggestion
     | Installing some apps and features with dependencies may also install additional apps and
       modules that are technically required, even if you won't actively use them.
```

File: content/applications/general.rst (hunk 2 above)
Comment (jcs-odoo) [reply content/applications/general.rst:31]: then, I add full stops at the end of each bullet point, that's more correct ^^

File: content/applications/general.rst (hunk 1 above)
Comment (jcs-odoo) [reply content/applications/general.rst:19]: I took the text from the footer on odoo.com .
I'll leave it like that if it's not incorrect.

**Comment by jcs-odoo** [comment]
@xpl-odoo  changes done

File: content/applications/general.rst (hunk 3 above)
Comment (xpl-odoo) [reply content/applications/general.rst]: Perfect :)

**Review by xpl-odoo — APPROVED** [review]

**Review by Feyensv — COMMENTED** [review]
LGTM, only a little global remark on `:doc:` and `:ref:` commands.

File: content/applications/general.rst [hunk 8]
```diff
@@ -16,130 +16,70 @@ General
...
-  able to show your customer their Quotes in a web page. Even though you
-  might not need or use the Website itself, it is needed for the Online
-  Quotes feature to work properly.
+   :align: center
 
-* **Always test app installation/removal on a duplicate** (or on a
-  free trial database): that way you can know what other apps may
-  be required, etc. This will avoid surprises when uninstalling
-  or when receiving your invoices.
+To complete the uninstallation, click on *Confirm*
```
Comment (AntoineVDV) [inline content/applications/general.rst:79]:
```suggestion
To complete the uninstallation, click on *Confirm*.
```

**Review by AntoineVDV — COMMENTED** [review]
LGTM

File: content/applications/general.rst [hunk 9]
```diff
@@ -16,130 +16,70 @@ General
...
+<sales/crm>`, :doc:`eCommerce <websites/ecommerce>`, :doc:`accounting <finance/accounting>`,
+:doc:`inventory <inventory_and_mrp/inventory>`, :doc:`point of sale <sales/point_of_sale>`,
+:doc:`project management <services/project>`, etc. Its unique value proposition is to be at the same
+time *very easy* to use and *fully integrated*.
+
+This section of the documentation covers all functional aspects that aren't app-specific. You can
+find, for example, information about:
+
+- :doc:`Users settings <general/users>` and their :doc:`access rights
+  <general/users/access_rights>`.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/general.rst:29]: right, j'ai changé tout cela et mis un petit texte en plus pour les autres points.

File: content/applications/general.rst [hunk 10]
```diff
@@ -16,130 +16,70 @@ General
...
-+----------------+----------------+
-
-In your applications' dashboard, you will see all the icons of your
-applications. Click on the application you want to uninstall. Then, on
-the form of the application, click on **Uninstall**.
+**Odoo** is a suite of open source business apps that cover all your company needs: :doc:`CRM
+<sales/crm>`, :doc:`eCommerce <websites/ecommerce>`, :doc:`accounting <finance/accounting>`,
+:doc:`inventory <inventory_and_mrp/inventory>`, :doc:`point of sale <sales/point_of_sale>`,
+:doc:`project management <services/project>`, etc. Its unique value proposition is to be at the same
+time *very easy* to use and *fully integrated*.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/general.rst:23]: Agreed, it can later be moved to the applications page.
For now, let's keep it here :)

**Comment by jcs-odoo** [comment]
@AntoineVDV updated :)

**Pull request merged**